install/cluster-operator/050-ConfigMap-strimzi-cluster-operator.yaml

```diff
@@ -28,3 +28,6 @@ data:
     logger.zookeepertrustmanager.name = org.apache.zookeeper
     logger.zookeepertrustmanager.level = WARN
     logger.zookeepertrustmanager.additivity = false
+
+    # Keeps separate level for Netty logging -> to not be changed by the root logger
+    io.netty = INFO
```

Comment:
Really sorry for not raising it at already the first time, but I didn't noticed it ... I think the lines 22-25 and 27-30 already do the same for other classes. Should we use the same here as well?

Reply:
so do it like:

```
logger.netty.name = io.netty
logger.netty.level = INFO
logger.netty.additivity = false
``` ??

Reply:
No problem, the goal is to have it correct

Reply:
Yeah, that looks like the way it was done there.

Reply:
Okay, thanks @scholzj for everything